src/components/SQForm/SQFormDropdown.js

```diff
@@ -19,6 +20,14 @@ import {EMPTY_LABEL} from '../../utils/constants';
 const EMPTY_VALUE = '';
 const EMPTY_OPTION = {label: EMPTY_LABEL, value: EMPTY_VALUE};
 
+const useStyles = makeStyles({
+  selectHeight: {
+    '& .MuiSelect-selectMenu': {
+      height: '1.1876em'
```

Comment:
Can you convert this to a `px` value, please?

Reply:
I used the exact value that other MUI fields are using (from MUI directly). I figure it'd be best to match it exactly to ensure there not be any issues in the future.

Reply:
> I used the exact value that other MUI fields are using (from MUI directly). I figure it'd be best to match it exactly to ensure there not be any issues in the future.

Everything from MUI results in `px`. If we leave the unit off a value, MUI will assume `px`.

I don't know `em` as well as `rem` but that should equate to `14px`. 
Can we test if using `px` will result in a pixel perfect alignment?

Reply:
The internal style Mui uses is in `em` to make the height of the element scale with the font-size. I believe `1.1876em` would equate to `14px` with the current font-size, but if the font-size on the component is changed from the default value somewhere the height won't scale properly if it's set to `14px` explicitly.